### PR TITLE
add disclaimer

### LINKED ISSUE #1658: Add disclaimer field to Work
### Need
Editorial note about metadata to help the reader, e.g. "31 December of the year of publication was used, because the actual publication date is unavailable."

### Changes

- [x] Add `disclaimer` field to Work model
- [x] Add `disclaimer` to work form
- [x] Add disclaimer to work overview
- [x] Add to bulk importer
- [x] Add to bulk updater
- [x] Show on coverpage of all documents
- [ ] Update https://docs.laws.africa/managing-works/bulk-imports-spreadsheet

### PR DESCRIPTION
**Pull request by goose-life**
closes #1658

### CODE REVIEW

File: indigo_app/templates/indigo_api/work_overview.html [hunk 1]
```diff
@@ -185,6 +185,15 @@ <h4 class="text-muted">{{ work.frbr_uri }}</h4>
                         {{ work.consolidation_note }}
                       </dd>
                     {% endif %}
+
+                    {% if work.disclaimer %}
+                      <dt>
+                        {% trans 'Disclaimer' %}
+                      </dt>
+                      <dd>
+                        {{ work.disclaimer|linebreaks }}
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_overview.html:194]: How will the renderer handle linebreaks?

Comment (goose-life) [reply indigo_app/templates/indigo_api/work_overview.html:194]: do you mean PDF?

Comment (goose-life) [reply indigo_app/templates/indigo_api/work_overview.html:194]: HTML
image


PDF
image

Comment (goose-life) [reply indigo_app/templates/indigo_api/work_overview.html:194]: I might totally be misunderstanding the question :sweat_smile:

Comment (longhotsummer) [reply indigo_app/templates/indigo_api/work_overview.html:194]: if we handle linebreaks in one place, it must be handled the same way everywhere (html, pdf, overview page, etc.). I'm just checking that we do that.

Comment (goose-life) [reply indigo_app/templates/indigo_api/work_overview.html:194]: The only difference seems to be that the PDF doesn't render blank linebreaks. Should we add support for that now?

**Comment by longhotsummer** [comment]
Simplest is no linebreaks -- do we strictly need them right now?

**Comment by goose-life** [comment]
The content team hadn't yet worded their first disclaimer when we spoke yesterday; I'll remove linebreaks for the time being.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**